ci-operator/step-registry/openshift/microshift/infra/pmlogs/openshift-microshift-infra-pmlogs-ref.yaml

```diff
@@ -1,6 +1,8 @@
 ref:
   as: openshift-microshift-infra-pmlogs
   from: test-runtime
+  best_effort: True
+  timeout: 10m
```

Comment:
It'd also be a good idea, but I still think in general, this script shouldn't register a failure in PR if the scp command fails due to for instance network connectivity.  This will not prevent the logs from reporting failures, they just won't count against the job.